addons/mail/static/src/widgets/messaging_menu/messaging_menu.js

```diff
@@ -42,9 +42,9 @@ const MessagingMenu = Widget.extend({
 const activityMenuIndex = SystrayMenu.Items.findIndex(SystrayMenuItem =>
     SystrayMenuItem.prototype.name === 'activity_menu');
 if (activityMenuIndex > 0) {
-    SystrayMenu.Items.splice(activityMenuIndex, 0, MessagingMenu);
+    SystrayMenu.Items.splice(activityMenuIndex, 0, MessagingMenuWidget);
 } else {
-    SystrayMenu.Items.push(MessagingMenu);
+    SystrayMenu.Items.push(MessagingMenuWidget);
 }
 
-export default MessagingMenu;
+export default MessagingMenuWidget;
```

Comment:
Could export immediately on `const MessagingMenuWidget`

(`export const ...`)

Reply:
that doesn't work on widget.extend

Reply:
I meant `export const MessagingMenuWidget = Widget.extend()` which works

Reply:
BTW here it still uses `export default`. I don't think there are issues in using explicit export instead, right?

Reply:
I ll retest when this is stabilized.